src/main/java/uk/gov/hmcts/reform/pip/model/publication/ListType.java

```diff
@@ -108,7 +108,34 @@ public enum ListType {
                                      false, true),
     RPT_SOUTHERN_WEEKLY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
         RPT_LONDON_WEEKLY_HEARING_LIST, "Residential Property Tribunal: Southern Region Weekly Hearing List",
-                                     false, true);
+                                     false, true),
+    SSCS_MIDLANDS_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        null, "Midlands First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+                                   false, true),
+    SSCS_SOUTHEAST_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        SSCS_MIDLANDS_DAILY_HEARING_LIST,
+        "South East First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+    false, true),
+    SSCS_WALES_AND_SOUTHEAST_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        SSCS_MIDLANDS_DAILY_HEARING_LIST,
+        "Wales and South West First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+    false, true),
+    SSCS_SCOTLAND_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        SSCS_MIDLANDS_DAILY_HEARING_LIST,
+        "Scotland First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+        false, true),
+    SSCS_NORTHEAST_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
```

Comment:
I'd suggest splitting out the NORTHWEST, NORTHEAST and SOUTHEAST parts of the list types into separate words (NORTH_WEST etc), as pub services will convert these into a single word (Northwest) to be displayed on the email template instead of splitting them out

Reply:
Thanks,  also updated data management and frontend PRs as well.